package.json

```diff
@@ -59,7 +59,7 @@
   "devDependencies": {
     "@gnosis.pm/mock-contract": "^3.0.8",
     "@gnosis.pm/owl-token": "^3.1.0",
-    "@gnosis.pm/solidity-data-structures": "^1.1.0",
+    "@gnosis.pm/solidity-data-structures": "^1.2.1",
```

Comment:
In the other PR (https://github.com/gnosis/solidity-data-structures/pull/8/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3) it says 1.2.0

Reply:
no longer